### DISCUSSION TITLE
Port to open?

### DISCUSSION THREAD

**Discussion by Btje1977** [hassio-addons/addon-unifi#358] [Q&A]
I tried forwarding 8443 from outside to my internal router and then towards my ha machine. The wierd thing is that I end up in my router login page when I try to setup the plugin. I have a kpn router and then a asus router placed all with their own ip range.

Which port should be open?

**Comment by frenck** [comment]
I really don't know how to answer this question, as it is not related to the add-on at all.

If anything: Don't expose these things to the outside world at all.

**Reply by similair** [reply]
hi @frenck I think @Btje1977 might mean the webui, it takes you to https://????:8443 and doesnt open the interface

**Reply by frenck** [reply]
That has nothing to do with ports to open, hence I don't understand the question he asked.

**Comment by Btje1977** [comment]
In got  it working somehow by forwarding the port to internal and after a reboot the interface for configuration started working.

**Reply by frenck** [reply]
> by forwarding the port to internal 

That should not be needed and I strongly recommend you not to expose ports to the internet for internal services like these.

**Reply by Btje1977** [reply]
Ok, I will close it and see if it keeps working.

**Reply by similair** [reply]
As @frenck says, delete that port forward.
Test it - it should have stopped working.
now in your browser type
ip_address_that_your_home_assistant_is_on:8443 enter
mine is 192.168.0.24:8443
if you get a warning, ignore it and continue.
